### ISSUE TITLE
The size of Grafana Monitor is so different with the size shown in minio.

### ISSUE THREAD

**Issue by ggservice007** [juicedata/juicefs#3492]
# In grafana
![image](https://user-images.githubusercontent.com/9192908/232760204-5df8746f-f5a1-4702-b131-1a368919e0a8.png)

# In mino
![image](https://user-images.githubusercontent.com/9192908/232760295-a416f345-72e0-4ce8-87c3-7694bb9c8858.png)

10x different

**Comment by solracsf** [comment]
If it helps:
https://juicefs.com/docs/community/faq/#why-is-file-system-data-size-different-from-object-storage-usage

**Comment by ggservice007** [comment]
@solracsf  thans a lot.

Has JuiceFS  performce lost for reading and writting if enable compression?

**Comment by SandyXSD** [comment]
> Has JuiceFS performce lost for reading and writting if enable compression?

Yes, but only a bit. You'd better test it by yourself to know the real difference in your setup.

**Comment by jyialive** [comment]
The usedSpace and totalInodes status of JuiceFS is not accurate as they are asynchronously calculated.
